src/controllers/IllnessStartDateController.ts

```diff
@@ -41,10 +41,12 @@ export class IllnessStartDateController extends BaseController<FormBody> {
         if (!illness) {
             return {};
         }
-        const startDate: Date = new Date(illness.illnessStart);
-        const day: string = startDate.getDate().toString();
-        const month: string = (startDate.getMonth() + 1).toString();
-        const year: string = startDate.getFullYear().toString();
+
+        const dateIngredients = illness.illnessStart.split('-', 3);
+
+        const year: string = dateIngredients[0];
+        const month: string = dateIngredients[1];
+        const day: string = dateIngredients[2];
```

Comment:
An even cleaner way of doing this would be:
`const [year, month, day] = value.split('-', 3);`